config/config.sample.php

```diff
@@ -715,9 +715,11 @@
 'versions_retention_obligation' => 'auto',
 
 /**
+ * Save additional metadata for versions
  * Save additional metadata (author, version tag, etc.) of each version of uploaded and edited files.
  *
- * WARNING: This feature CANNOT be temporarily disabled once enabled. Consequitive enabling would require repair job that erases all extended versions metadata.
+ * WARNING: This feature CANNOT be temporarily disabled once enabled.
+ * Consequitive enabling would require repair job that erases all extended versions metadata.
```

Comment:
```suggestion
 * Consecutive enabling would require repair job that erases all extended versions metadata.
```
This just fixes the spelling. I will merge that.